<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_management</class>
 <widget class="QDialog" name="user_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>486</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="4">
    <widget class="QLineEdit" name="lineMoney"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="lineName"/>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Money</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Card ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineID"/>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QTableWidget" name="userView"/>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="readCard">
     <property name="text">
      <string>Read card</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="add_user">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="clear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QCheckBox" name="checkSuper">
     <property name="text">
      <string>Super</string>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Käyttäjien hallinta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
